### PR TITLE
Add interactive password input

### PR DESCRIPTION
**Pull request by svagner**
Interactive input password can be very convenient and more secure:)

### CODE REVIEW

File: go/cmd/ccql/main.go [hunk 1]
```diff
@@ -35,6 +37,7 @@ func main() {
 	help := flag.Bool("help", false, "Display usage")
 	user := flag.String("u", osUser, "MySQL username")
 	password := flag.String("p", "", "MySQL password")
+	askPassword := flag.Bool("P", false, "Ask mySQL password")
```
Comment (shlomi-noach) [inline go/cmd/ccql/main.go]: Can we please change argument to `ask-pass`? This will make it compatible with `gh-ost` and with Percona Toolkit.

Comment (svagner) [reply go/cmd/ccql/main.go]: Sure. Will change arg

**Comment by shlomi-noach** [comment]
Long overdue opportunity to set up TravisCI for this repo, adding public visibility into CI

**Comment by shlomi-noach** [comment]
Added `golang.org/x/crypto/ssh/terminal` dependency to make this build pass

**Comment by shlomi-noach** [comment]
Thank you

**Pull request merged**